client/src/routes/settings/+page.svelte

```diff
@@ -194,9 +256,107 @@
     </Dialog>
   {/if}
 
-  <Button on:click={() => (showDeletionPrompt = true)}>
-    <Label>Delete Account</Label>
-  </Button>
+  <p id="email">Email: {$user?.email == null ? "Anonymous User" : $user.email}</p>
```

Comment:
consider using `??`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
resolved in 3ee8bb956cde4dc4ea1245bf9c6cb09f60c1b3b2